### ISSUE TITLE
Meeting

### ISSUE THREAD

**Issue by jbesraa** [CYPIAPT-LNDSE/sdm-app#4]
1 Feb at 16:00
@denesnori  @njsfield @rosatown

**Comment by denesnori** [comment]
@rosatown and @DHayes1987 is it ok to meet on Thursday (sometime before the pitch) instead of Wednesday  at Anna Freud? We want to be confident that we can show you a fully functioning prototype, and practice our pitch together. :)

**Comment by rosatown** [comment]
Sure -- are you wanting us to come to Bethnal Green? If so, we could come by between 10:30 AM and 11:30 AM if that suits you

**Comment by denesnori** [comment]
We would come to your office or to Anna Freund. What about 3.00 or 4.00 pm, just before the pitch on Thursday?

**Comment by rosatown** [comment]
I think that the pitch is next Thursday (the 9th) -- is that what you're talking about?

**Comment by rosatown** [comment]
Ah I see now that the 2nd is a "casual update" on progress and the 9th is the official pitch. Would you like to meet at the Anna Freud then this Thursday (2nd) at 4:00 PM?

**Comment by njsfield** [comment]
Ah sorry for the confusion! We should of clarified the two dates- yes Thursday at 4pm would be great if you can find time :)

**Comment by rosatown** [comment]
Yes that's perfect, and we have the time for sure. See you then!

**Comment by rosatown** [comment]
And no worries -- we were confused too about the timing but now it all makes sense. :)

**Comment by njsfield** [comment]
Hi @rosatown @DHayes1987, just checking to make sure we're okay to meet today at 16:00 still? Let us know if you have any problems/delays and need adjust time :)

**Comment by DHayes1987** [comment]
This is fine. We look forward to seeing you then!!!

Best wishes,

________________________________
Daniel Hayes
Researcher in Shared Decision Making
Project Manager for i-THRIVE Option Grid Decision Aids
MSc Research Supervisor for Psychoanalytic Developmental Psychology
4-8 Rodney Street, London, N1 9JH
Tel +44 (0) 20 7443 2220

[cid:image001.jpg@01D27D41.E1EE4200]  [cid:image002.png@01D27D41.E1EE4200]

**Comment by rosatown** [comment]
Yes this time still works great for us. We are on the 1st floor so just buzz EBPU when you arrive and someone will let you in. Looking forward to seeing you soon!

**Comment by jbesraa** [comment]
@rosatown @DHayes1987  We are very sorry. we will be 30 minutes late. apologies.

in the meantime, enjoy the prototype: https://cypiapt-lndse.github.io/sdm-app/

**Comment by rosatown** [comment]
Not to worry--it will probably be just Dan there when you get there but I will be arriving shortly. 

Prototype is looking really cool--are we able to make some minor changes to wording when we meet up?

**Comment by DHayes1987** [comment]
No probs,

See you soon

Best wishes,

________________________________
Daniel Hayes
Researcher in Shared Decision Making
Project Manager for i-THRIVE Option Grid Decision Aids
MSc Research Supervisor for Psychoanalytic Developmental Psychology
4-8 Rodney Street, London, N1 9JH
Tel +44 (0) 20 7443 2220

[cid:image001.jpg@01D27D6C.9E280AD0]  [cid:image002.png@01D27D6C.9E280AD0]
